catkit/emulators/ZwoCamera.py

```diff
@@ -79,6 +82,10 @@ def set_image_type(self, image_type):
     def capture(self, initial_sleep=0.01, poll=0.01, buffer=None, filename=None):
         pass
 
+    @abc.abstractmethod
+    def capture_video_frame(self, buffer=None, filename=None, timeout=None):
+        return self.capture(buffer=buffer, filename=filename)
```

Comment:
👍

Reply:
Yep, you were right after all. This doesn't need to be in hicat-package.